lms/djangoapps/certificates/models.py

```diff
@@ -549,11 +549,11 @@ class CertificateHtmlViewConfiguration(ConfigurationModel):
             "default": {
                 "url": "http://www.edx.org",
                 "logo_src": "http://www.edx.org/static/images/logo.png",
-                "logo_alt": "Valid Certificate"
+                "logo_alt": "Home"
```

Comment:
I think this should be "edX Home Page"  
applies to the honor cert below and to the test files.